### PR TITLE
Persist used tricks

### PR DESCRIPTION
**Pull request by henriquegemignani**
It's a bit slow to calculate, so let's load it directly.

### CODE REVIEW

File: randovania/games/am2r/json_data/header.json [hunk 1]
```diff
@@ -4296,6 +4296,92 @@
             "to_shuffle_proportion": 1.0
         }
     },
+    "used_trick_levels": {
```
Comment (Miepee) [inline randovania/games/am2r/json_data/header.json:4299]: Does this mean, that as the DB is maturing, that i need to manually add the trick levels here?

Comment (henriquegemignani) [reply randovania/games/am2r/json_data/header.json:4299]: No, this is updated automatically when you save.

Comment (henriquegemignani) [reply randovania/games/am2r/json_data/header.json:4299]: I thought of making this only show up in the binary version of the db. Could still do that.

Comment (Miepee) [reply randovania/games/am2r/json_data/header.json:4299]: Ah okay, must've missed it.

**Review by ThanatosGit — APPROVED** [review]

**Pull request merged**